yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlRunner.java

```diff
@@ -65,7 +67,24 @@ public final class YamlRunner {
     private final YamlExecutionContext executionContext;
 
     public interface YamlConnectionFactory {
+        /**
+         * Convert a connection uri into an actual connection.
+         * @param connectPath the path to connect to
+         * @return A new {@link RelationalConnection} for the given path appropriate for this test class
```

Comment:
Add "...and an empty set if we only support the current version?"?

Reply:
I'm assuming you meant to put this comment in response to the javadoc on `getVersionsUnderTest` (line 85)